pkg/kubelet/server/stats/handler.go

```diff
@@ -28,7 +28,7 @@ import (
 	cadvisorapi "github.com/google/cadvisor/info/v1"
 	"k8s.io/klog"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
This change is unnecessary.

Reply:
Yes, I think that was made by go imports. I will remove it